File: url-shortener/android/app/src/main/java/com/nubank/urlshortener/view/UrlShortenerActivity.java
package com.nubank.urlshortener.view;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.nubank.urlshortener.R;

public class UrlShortenerActivity extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_url_shortener);
    }
}

File: url-shortener/android/app/src/main/java/com/nubank/urlshortener/view/UrlShortenerActivity.kt
package com.nubank.urlshortener.view

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.nubank.urlshortener.R

class UrlShortenerActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_url_shortener)
    }
}